examples/cvae/baseline.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, masked_with=-1):
 
     def forward(self, input, target):
         target = target.view(input.shape)
-        loss = F.binary_cross_entropy(input, target, reduction="none")
-        loss[target == self.masked_with] = 0
+        # only calculate loss on target pixels (value = -1)
+        loss = F.binary_cross_entropy(input[target != self.masked_with], target[target != self.masked_with], reduction='none')
```

Comment:
Could you run `make lint` to fix the lint error? I think it's just a line too long.

Reply:
Had checked the format using `make lint` and fix the lint error.